src/position.cpp

```diff
@@ -0,0 +1,63 @@
+#include "position.h"
+
+namespace openmc {
+
+//==============================================================================
+// Position implementation
+//==============================================================================
+
+Position&
+Position::operator+=(Position other)
```

Comment:
A const reference (`const Position &other`) might be good here to avoid the memory copy.

There are places external to `Position` methods where this could be useful too.

Reply:
👍 Suggestion pulled. Reading more, it looks like this will only make a difference in certain cases for objects of this size.